### ISSUE TITLE
Test https://beta.oerworldmap.org by @formimkontext

### ISSUE THREAD

**Issue by literarymachine** [hbz/oerworldmap-ui#190]
# [Landing Page](https://beta.oerworldmap.org)
- [x] I can click on all links

# [User registration](https://beta.oerworldmap.org/user/register)
- [x] I can register a user
- [x] I receive the verification email and can click the link
- [x] I can log in with the new user
- [x] I can log back out
- [x] I can reset the password by entering the email
- [x] I can log in with the new password sent by email
- [x] I can change the new password to an own one
- [x] I can log in with the new password I supplied
- [ ] I can administer groups (requires admin privileges)

# [Map / List](https://beta.oerworldmap.org/resource/)
- [x] I can move around on the map
- [x] I can click on large clusters of pins to zoom in
- [x] I can click on small clusters of pins to get a list popup
- [x] I can click on individual pins
- [x] I can show the list by clicking on the list icon in the search box
- [ ] I can set and unset filters
- [x] I can show and hide the extended filter section
- [ ] I can share, export and print the list
- [ ] I can add one entry of each type
- [x] I can show the global statistics
- [x] I can list the recent additions
- [x] I can list the likes
- [x] I can list the lighthouses

# [Individual items](https://beta.oerworldmap.org/resource/urn%3Auuid%3A29056e88-727b-407e-b00a-16df279ac506)
- [x] I can find the data I expect
- [ ] I can like and unlike the item
- [x] I can add a lighthouse
- [x] I can set the `I'm a member` relation for organizations
- [x] I can set the `I'm attending` relation for events
- [x] I can add a comment
- [x] I can edit and save the entry
- [ ] I can view the log of the entry (requires admin privileges)
- [ ] I can delete comments (requires admin privileges)
- [ ] I can delete lighthouses (requires admin privileges)

**Comment by formimkontext** [comment]
- i can not unset a main type filter just by clicking on it. i have to click ›clear filter‹ instead.
- sharing a permalink (https://beta.oerworldmap.org/resource/?filter.about.@type=Person&filter.about.location.address.addressCountry=BR&size=20&sort=dateCreated%3ADESC&features=true) by clicking on the icons in the sharing-modal, does not copy the entire URL (shared link is: https://beta.oerworldmap.org/resource/?filter.about.@type=Person). i have to copy the entire link manually to share it.
- i can not read the error report for editing the ›date created‹ form (should match pattern "^([\+-]?\d{4}(?!\d{2}\b))((-?)((0[1-9]|1[0-2])(\3([12]\d|0[1-9]|3[01]))?|W([0-4]\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\d|[12]\d{2}|3([0-5]\d|6[1-6])))([T\s]((([01]\d|2[0-3])((:?)[0-5]\d)?|24\:?00)([\.,]\d+(?!:))?)?(\17[0-5]\d([\.,]\d+)?)?([zZ]|([\+-])([01]\d|2[0-3]):?([0-5]\d)?)?)?)?$") 
- can not add a story / event because i don’t know how to enter a proper date. miss the cancel-button within the ›Add Story‹-modal
- i can not unlike the item (authentification fails)

**Comment by literarymachine** [comment]
> i can not unset a main type filter just by clicking on it. i have to click ›clear filter‹ instead

Confirmed, see hbz/oerworldmap-ui#200

> sharing a permalink by clicking on the icons in the sharing-modal, does not copy the entire URL

Confirmed, see hbz/oerworldmap-ui#201

> i can not read the error report for editing the ›date created‹ form

True, the errors are sometimes still hard to parse (for a human). We will be working on this. Until it is done, would adding a placeholder be enough (`YYYY-MM-DD`)?

> can not add a story / event because i don’t know how to enter a proper date.

See above.

> miss the cancel-button within the ›Add Story‹-modal

Confirmed, see hbz/oerworldmap-ui#202

> i can not unlike the item (authentification fails)

Confirmed, see hbz/oerworldmap-ui#199

**Comment by formimkontext** [comment]
> True, the errors are sometimes still hard to parse (for a human). We will be working on this. Until it is done, would adding a placeholder be enough (YYYY-MM-DD)?

Yes, a placeholder (YYYY-MM-DD) would be great.

**Comment by literarymachine** [comment]
> Yes, a placeholder (YYYY-MM-DD) would be great.

See #237.

**Comment by formimkontext** [comment]
some more issues regarding the new ui: 
- order of dates in calender view ist strange: order of months and days is divergent
- add "close" or "quit" button at the bottom of every modal (e.g. global statistics) or form  
- like button: change icon to star instead of thumb   
- disable automatic map zoom via scrolling / trackpad in forms 
- login: wording inconsistently: the popover should ask for "Email / Password" instead for "Username / Password"
